adserver/views.py

```diff
@@ -956,6 +958,110 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class AdvertiserAuthorizedUsersView(
+    AdvertiserAccessMixin, UserPassesTestMixin, ListView
+):
+
+    """Authorized users for an advertiser."""
+
+    model = get_user_model()
+    template_name = "adserver/advertiser/users.html"
+
+    def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
+        context = super().get_context_data(**kwargs)
+        context.update({"advertiser": self.advertiser, "users": self.object_list})
```

Comment:
Any reason we're setting `users` instead of `context_object_name = users`? 

https://docs.djangoproject.com/en/3.2/topics/class-based-views/generic-display/#making-friendly-template-contexts

Reply:
I didn't know about that. That's awesome.